ExampleApp/package.json

```
@@ -0,0 +1,49 @@
+{
+  "name": "AwesomeProject",
+  "version": "0.0.1",
+  "private": true,
+  "scripts": {
+    "android": "react-native run-android",
+    "ios": "react-native run-ios",
+    "lint": "eslint .",
+    "start": "react-native start",
+    "test": "jest"
+  },
+  "dependencies": {
+    "@react-native/new-app-screen": "0.81.0",
+    "react": "19.1.0",
+    "react-native": "0.81.0",
```

Let's keep this consistent between the SDK & the example app:
```suggestion
    "react": "^19.1.0",
    "react-native": "^0.81.0",
```

---

This was not addressed

---

I think we should commit package-lock.json for the example project as well

---

> I think we should commit package-lock.json for the example project as well
After discussion, the solution is to keep only yarn.lock